bin/development-server.js

```diff
@@ -61,9 +54,12 @@ app.use(express.static("public/js/test/examples"));
 app.use(express.static("public"));
 
 // Routes
-
 app.get("/", function(req, res) {
-  res.sendFile(path.join(__dirname, "../index.html"));
+  const tpl = fs.readFileSync(path.join(__dirname, "../index.html"));
+  const html = _.template(tpl)({
```

Comment:
Nice. I'd very much prefer to pull in individual lodash methods though, so just require `lodash-template`. I don't see us using much of lodash, so it'd be great to only pull in the methods we need.